templates/part.board.mainView.php

```diff
@@ -5,12 +5,12 @@
 		<p>{{ statusservice.text }}</p></div>
 </div>
 
-<div id="board-header">
-	<div id="app-navigation-toggle" class="icon-menu" ng-click="navibar.show=!navibar.show"></div>
-	<a class="crumb" href="#" title="<?php p($l->t('All Boards')); ?>">
-		<i class="icon icon-home"></i>
-		<span class="hidden-visually"><?php p($l->t('All Boards')); ?></span>
-	</a>
+<div id="controls">
```

Comment:
"same code as files" I remember that discussion from a different PR :wink: files also has a wrong. This is what is causing issue with positioning of icons mentioned by @juliushaertl